assets/less/zenodo-rdm/collections/menu.overrides

```diff
@@ -0,0 +1,10 @@
+/*
+ *   Copyright (C) 2021-2022 CERN.
+ *
+ * Invenio App RDM is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+#invenio-nav.ui.menu .ui.menu .item a:not(.ui.button) {
+  padding: 2.8rem 1rem;
```

Comment:
does it make sense to add variables ?

Reply:
It's just to fill the height of the header, so I don't think they should be needed anywhere else